iv-outlier-detection/src/genesisVolatility.ts

```diff
@@ -0,0 +1,48 @@
+import { Requester } from '@chainlink/external-adapter'
```

Comment:
This code is duplicated. We should reuse the `genesisVolatility` adapter we already have.

Reply:
It seems to me that this is the only outstanding item right now. Are we good to use this as-is and use the `genesisVolatility` adapter once we move this to a composite adapter, or should we publish that package and deal with that now?